coral/src/app/features/topics/details/overview/index.tsx

```diff
@@ -1,9 +0,0 @@
-import { useTopicDetails } from "src/app/features/topics/details/TopicDetails";
```

Comment:
why is this file deleted? I mean, yeah, it's only a wrapper :D but we used that pattern of: "have a wrapper index.tsx in router" in all other views we build with tabs, too. So `pages/` is a representation of our routing.

Reply:
You're absolutely right! The github diff view lead me to confusion 🙈